GameDefinition/Content/ItemKindEmbed.hs

```diff
@@ -32,7 +32,7 @@ stairsUp = ItemKind
   , iaspects = []
   , ieffects = [Ascend True]
   , ifeature = [Identified, Durable]
-  , idesc    = ""
+  , idesc    = "Stairs that rise towards escape."
```

Comment:
Hmm, to be honest, this is never displayed, not even in the ~ menu. :) These are the pseudo-items that reside inside tiles and can be activated (e.g., activating stairs ascends levels, activating stair trap causes some mishap, etc.). Moreover, to my surprise, there is no description of tiles. Perhaps that's for the best? Perhaps I could just display the embedded item descriptions for the tiles that have them and keep the other tiles silent (a granite wall is a granite wall is a granite wall)?

Let's merge these descriptions in and revisit them once we can experiment how they are shown and in what context. Could you open an issue about showing these in the ~ menu and another one about showing them when crosshair points at a tile? Thank you!